src/util/util.js

```diff
@@ -0,0 +1,74 @@
+import _ from 'lodash';
```

Comment:
this file should be named `util/environment.js` or something rather than just the generic `util.js`